api/v1alpha3/types.go

```diff
@@ -78,6 +78,11 @@ type VirtualMachineCloneSpec struct {
 	// +optional
 	Server string `json:"server,omitempty"`
 
+	// Thumbprint is the thumbprint of the given Server
+	// when supplied Insecure should not be set to true
```

Comment:
This description should be adapted to the machine object (e.g. there is no insecure at the machine level)

Reply:
ack